packages/@ama-sdk/core/cli/update-spec-from-npm.cts

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+
+/*
+ * Update the OpenAPI spec from an NPM package
+ */
+
+import { existsSync } from 'node:fs';
+import { copyFile, readFile } from 'node:fs/promises';
+import type { OpenApiToolsConfiguration, OpenApiToolsGenerator } from '../src/fwk/open-api-tools-configuration';
+
+void (async () => {
+  const packageName = process.argv[2];
+  const packagePath = process.argv[3];
+
+  if (!packageName || !packagePath) {
+    console.error('Need to provide packageName and packagePath\nUsage: amasdk-update-spec-from-npm <package-name> <package-path>');
+    process.exit(-1);
+  }
+
+  const specSourcePath = require.resolve(`${packageName}/${packagePath}`);
+  const openApiConfigPath = './openapitools.json';
```

Comment:
this can technically be an input on the openapi tools script

Reply:
I added an option to specify the output path